package.json

```diff
@@ -51,7 +51,7 @@
     "wd": "^0.4.0"
   },
   "devDependencies": {
-    "jasmine-node": "~1",
-    "jshint": "^2.6.0"
+    "jshint": "^2.6.0",
+    "appium": "1.5.3"
```

Comment:
Yep, we can, I just thought it was a bit awkward. No problem, I can put it back.

Can you explain how using this method saves time within the CI context, but adding as a dependency does not? I will add a bit of in-line documentation around it just to explain why we're doing it this way.